zeppelin-web/src/app/visualization/builtins/visualization-barchart.js

```diff
@@ -57,6 +59,29 @@ export default class BarchartVisualization extends Nvd3ChartVisualization {
 
   configureChart(chart) {
     var self = this;
+    // prevent to overlap in text if using multiBarChart
+    // if size of xLabel are over maxRotate, the MutiBarChart don't show text
+    var minRotate = 30;
+    var maxRotate = 80;
+    var xLabelsLength = Object.keys(self.xLabels).length;
```

Comment:
You're right, I missed when moving the code!
Let me check and update again, Thanks @1ambda